GVFS.sln

```diff
@@ -29,7 +29,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GVFS.Common", "GVFS\GVFS.Co
 		{A4984251-840E-4622-AD0C-66DFCE2B2574} = {A4984251-840E-4622-AD0C-66DFCE2B2574}

```

Comment:
This isn't showing up in any of your diffs, so I'm leaving it here.

In a few places, the fastfetch code does "git to Windows" and "Windows to git" conversions. It appears that those code paths are doing the right thing since they're mostly actually doing "git to .NET" and ".NET to git", and the .NET code is platform-aware. But please review all of those instances, confirm that they are correct, and rename them as appropriate.

The ones I have found are:
* `Index.ToWindowsTime` and `ToUnixTime`
* `FromGitRelativePathToWindowsFullPath` and `FromWindowsFullPathToGitRelativePath`, both called from `Index.cs`

That was just from a quick search for the word "Windows" in the fastfetch code. A more thorough search would be helpful as well.